Firebase/Core/Private/FIRAppInternal.h

```diff
@@ -134,6 +134,22 @@ typedef NSString *_Nullable (^FIRAppGetUIDImplementation)(void);
  */
 + (BOOL)isDefaultAppConfigured;
 
+
+/**
+ * Registers a given third-party library with the given version number to be reported for
+ * analyitcs.
+ *
+ * @param library Name of the library
+ * @param version Version of the library
+ */
++ (void)registerLibrary:(nonnull NSString *)library withVersion:(nonnull NSString *)version \
```

Comment:
Nit: the `\` shouldn't be necessary here.

Reply:
Done.